### DISCUSSION TITLE
So are repositories hosting DDoS and hacking scripts allowed?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23658] [New to GitHub]
<p>I’ve seen someone hosting Hacking scripts on Github, my question is are these content allowed on Github?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/bayodino">@BayoDino</a>,</p>
<p>Thanks for being here! In general, according to our terms of service:</p>
<p><strong>Short version:</strong></p>
<p>Your use of the Website and Service <strong>must not violate any applicable laws, including copyright or trademark laws</strong>, export control or sanctions laws, or other laws in your jurisdiction. You are responsible for making sure that your use of the Service is in compliance with laws and any applicable regulations.</p>
<p>I hope that helps!</p>

**Comment by ghost** [comment]
<p>Thank you for the answer! and Sorry if I delay in responding.</p>
